cli/nodeutil.ts

```diff
@@ -19,7 +19,16 @@ export var pxtCoreDir: string = path.join(targetDir, "node_modules/pxt-core")
 
 export function setTargetDir(dir: string) {
     targetDir = dir;
-    pxtCoreDir = path.join(targetDir, "node_modules/pxt-core");
+    pxtCoreDir = path.join(targetDir, "node_modules", "pxt-core");
+
+    if (!fs.existsSync(pxtCoreDir)) {
+        // Fix for NPM 3: dependencies are flattened, which means pxt-core is not under [target]/node_modules/pxt-core.
+        // In that case, use the pxt-core that nodeutil.js script is running in. It's in pxt-core/built/nodeutil.js, so
+        // go up once to reach root of pxt-core.
+        pxtCoreDir = path.join(__dirname, "..");
```

Comment:
Wasn't sure about the best way to test that. Ended up looking for package.json and making sure "name" is "pxt-core".